pyscf/pbc/df/rsdf_helper.py

```diff
@@ -0,0 +1,1297 @@
+#!/usr/bin/env python
```

Comment:
Many functions in this file are needed only by this module. Can you prefix them with `_` to indicate they are private functions? For functions which may be used by other modules, please add docs for all arguments.

Reply:
fixed in [this commit](https://github.com/pyscf/pyscf/pull/1034/commits/94585b16b1987beda31e273b0e5e2bc003e76f01)